docs/copied-from-beats/security/securing-beats.asciidoc

```diff
@@ -1,33 +1,79 @@
 [role="xpack"]
 [[securing-beats]]
-== {beatname_uc} and {security}
+== Configure {beatname_uc} to use {security}
 
-If you want {beatname_uc} to connect to a cluster that has {security} enabled,
-there are extra configuration steps.
+++++
+<titleabbrev>Use {security}</titleabbrev>
+++++
 
+If you want {beatname_uc} to connect to a cluster that has
+{securitydoc}/xpack-security.html[{security}] enabled, there are extra
+configuration steps:
+
+. <<beats-basic-auth>>.
++
+ifeval::["{beatname_lc}"=="filebeat"]
+To send data to a secured cluster through the `elasticsearch` output,
+{beatname_uc} needs to authenticate as a user who can manage index templates,
+monitor the cluster, create indices, read and write to the indices
+it creates, and manage ingest pipelines. 
+endif::[]
+ifeval::["{beatname_lc}"!="filebeat"]
 To send data to a secured cluster through the `elasticsearch` output,
 {beatname_uc} needs to authenticate as a user who can manage index templates,
-monitor the cluster, create indices, and read, and write to the indices
-it creates. See <<beats-basic-auth>>.
+monitor the cluster, create indices, and read and write to the indices
+it creates. 
+endif::[]
 
-If encryption is enabled on the cluster, you also need to enable HTTPS in the
-{beatname_uc} configuration. See <<beats-tls>>.
+. <<beats-user-access>>.
++
+To search the indexed {beatname_uc} data and visualize it in {kib}, users need
+access to the indices {beatname_uc} creates.
 
-In addition to configuring authentication credentials for the {beatname_uc}
-itself, you need to grant authorized users permission to access the indices it
-creates. See <<beats-user-access>>.
+. <<beats-tls>>.
++
+If encryption is enabled on the cluster, you need to enable HTTPS in the
+{beatname_uc} configuration.
 
 ifeval::["{beatname_lc}"!="apm-server"]
-If you plan to monitor {beatname_uc} in Kibana, you must also 
-<<beats-system-user,configure the `beats_system` built-in user>>.
+. <<beats-system-user>>.
++
+{beatname_uc} uses the `beats_system` user to send monitoring data to {es}. If
+you plan to monitor {beatname_uc} in {kib} and have not yet set up the
+password, set it up now.
 endif::[]
 
 For more information about {security}, see
-{xpack-ref}/xpack-security.html[Securing {es} and {kib}].
+{xpack-ref}/xpack-security.html[Securing the {stack}].
+
+[float]
+=== {beatname_uc} features that require authorization
+
+After securing {beatname_uc}, make sure your users have the roles (or associated
+privileges) required to use these {beatname_uc} features. You must create the
+++{beat_default_index_prefix}_writer++ and
+++{beat_default_index_prefix}_reader++ roles (see <<beats-basic-auth>> and
+<<beats-user-access>>). The `machine_learning_admin` and `kibana_user` roles are
```

Comment:
does this show up or is behind a conditional somewhere? 
if it does: is the machine learning role relevant?

Reply:
It is but it's broken, fixing.